src/main/java/space/obminyashka/items_exchange/dto/UserChangePasswordDto.java

```diff
@@ -1,21 +1,18 @@
 package space.obminyashka.items_exchange.dto;
 
-import space.obminyashka.items_exchange.annotation.FieldMatch;
-import space.obminyashka.items_exchange.util.PatternHandler;
 import lombok.AllArgsConstructor;
-import lombok.Builder;
 import lombok.Getter;
 import lombok.NoArgsConstructor;
+import space.obminyashka.items_exchange.util.PatternHandler;
```

Comment:
You have just replaced the import and rewrote the commit history.

Reply:
As you that has came because I removed 2 additional imports from here and added 1 new.